### PR TITLE
#28719: Adding AI Generative postman tests and make necessary changes in prod code

### LINKED ISSUE #28719: Write Postman Tests for Generative AI Endpoints
### Parent Issue
https://github.com/dotCMS/core/issues/27973

### User Story
As a Java developer, I want to write Postman tests for the following Generative AI Endpoints:
- /api/v1/ai/text/generate (GET)
- /api/v1/ai/text/generate (POST)
- /api/v1/ai/image/generate (GET)
- /api/v1/ai/image/generate (POST)

### Acceptance Criteria
- Postman test collection includes tests for:
  - /api/v1/ai/text/generate (GET)
  - /api/v1/ai/text/generate (POST)
  - /api/v1/ai/image/generate (GET)
  - /api/v1/ai/image/generate (POST)
- Tests validate successful response status and structure.
- Tests include various valid and invalid input scenarios.
- Test results are documented and shared with the team.

### dotCMS Version
master

### Proposed Objective
Quality Assurance

### Proposed Priority
Priority 2 - Important

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.
- Slack Conversation: [link]
- Support Ticket: [link]
- API Documentation: [link]

### Assumptions & Initiation Needs
- Access to the Generative AI endpoints.
- Valid API keys or tokens for authentication.
- Sample input data for text and image generation.

### Quality Assurance Notes & Workarounds
- Ensure that tests cover a wide range of scenarios including edge cases.
- Verify that the API responses match the expected structure and content.
- Perform regression testing to ensure no existing functionality is broken by these tests.

### Sub-Tasks & Estimates
1. Review API documentation for Generative AI endpoints (2 hours)
2. Set up Postman environment and variables (2 hours)
3. Write tests for /api/v1/ai/text/generate (GET) (3 hours)
4. Write tests for /api/v1/ai/text/generate (POST) (3 hours)
5. Write tests for /api/v1/ai/image/generate (GET) (3 hours)
6. Write tests for /api/v1/ai/image/generate (POST) (3 hours)
7. Validate tests with various scenarios and inputs (4 hours)
8. Document test results and share with the team (2 hours)
9. Update documentation to reflect the new tests (1 hour)

Total Estimate: 23 hours

### PR DESCRIPTION
**Pull request by victoralfaro-dotcms**
Making necessary changes to execute postman tests for AI Generative endpoints. Also adding the actual tests.

PR fixes #28719

### CODE REVIEW

File: dotcms-postman/src/main/resources/postman/AI.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+		"name": "AI",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "11174695",
+		"_collection_link": "https://codigami.postman.co/workspace/dotcms2~bd48d5e8-9590-4fc6-96fc-68937fcf781b/collection/11174695-aa4d25f0-c62e-47ea-924c-d2f51d25e6db?action=share&source=collection_link&creator=11174695"
+	},
+	"item": [
+		{
+			"name": "Generative",
+			"item": [
+				{
```
Comment (dsilvam) [inline dotcms-postman/src/main/resources/postman/AI.postman_collection.json:13]: These postman tests are missing negative cases

**Review by dsilvam — DISMISSED** [review]
Include negative cases for the postman tests

**Review by freddyDOTCMS — APPROVED** [review]
According to the title, it looks like it is just postman test but we are including several changes in code here.
I think that maybe we should sent this changes on a second PR

File: dotcms-postman/src/main/resources/postman/AI.postman_collection.json (hunk 1 above)
Comment (victoralfaro-dotcms) [reply dotcms-postman/src/main/resources/postman/AI.postman_collection.json:13]: Negative postman tests added.

**Review by dsilvam — APPROVED** [review]

**Pull request merged**